DER/DerivativesContracts/Options.rdf

```diff
@@ -171,6 +209,20 @@
 		<owl:disjointWith rdf:resource="&fibo-der-der-opt;ConditionalPayoutCommitment"/>
 	</owl:Class>
 	
+	<owl:Class rdf:about="&fibo-der-der-opt;BondOption">
+		<rdfs:subClassOf rdf:resource="&fibo-der-der-opt;OptionOnFuture"/>
+		<rdfs:subClassOf rdf:resource="&fibo-der-sbd-sbd;DebtInstrumentDerivative"/>
+		<rdfs:subClassOf>
+			<owl:Restriction>
+				<owl:onProperty rdf:resource="&fibo-der-drc-bsc;hasUnderlier"/>
+				<owl:someValuesFrom rdf:resource="&fibo-sec-dbt-bnd;Bond"/>
+			</owl:Restriction>
+		</rdfs:subClassOf>
+		<rdfs:label xml:lang="en">bond option</rdfs:label>
+		<skos:definition xml:lang="en">option on a futures contract whose underlying asset is a bond</skos:definition>
```

Comment:
Not according to https://www.investopedia.com/terms/b/bondoption.asp which says it's an option on the bond not a future!! This is a major discrepancy. Maybe T Bills are a special case.